packages/astro/src/core/endpoint/index.ts

```diff
@@ -16,21 +16,24 @@ import { AstroError, AstroErrorData } from '../errors/index.js';
 import { warn, type LogOptions } from '../logger/core.js';
 import { callMiddleware } from '../middleware/callMiddleware.js';
 import { isValueSerializable } from '../render/core.js';
+import { simpleEndpointOutputToResponse } from '../util.js';
```

Comment:
We can't import `util` in this file because this file relies on `node:` specific utilities. It's preferable to move `simpleEndpointOutputToResponse` somewhere else, where `node` is not used.